src/constantGuards.ts

```diff
@@ -1,13 +1,30 @@
 import {ReasonGuard} from './ReasonGuard';
 import {checkerToGuard} from './Checker';
+import {NegatableGuard} from './NegatableGuard';
 
-const trueGuard: ReasonGuard<unknown, unknown> =
+const trueGuard: NegatableGuard<unknown, unknown, never> =
 	checkerToGuard(() => 'true');
 
-const falseGuard: ReasonGuard<unknown, never> =
+const falseGuard: NegatableGuard<unknown, never, unknown> =
 	checkerToGuard(() => {
 		throw new Error('false');
 	});
 
-export const constantGuards: (result: boolean) => ReasonGuard<unknown, unknown> =
+export const constantGuards: (result: boolean) => NegatableGuard<unknown, unknown, unknown> =
 	(result: boolean) => result ? trueGuard : falseGuard;
+
+const unnegatableTrueGuard: ReasonGuard<unknown, unknown> =
+	// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
```

Comment:
I think it will tolerate the unused params without the suppress if you name them `_something`

Reply:
Nope! Tried that.